cnv/cnv_virtual_machines/cnv_vm_networking/virt-using-mac-address-pool-for-vms.adoc

```diff
@@ -0,0 +1,18 @@
+[id="virt-using-mac-address-pool-for-vms"]
+= Using a MAC address pool for virtual machines
+include::modules/cnv-document-attributes.adoc[]
+:context: virt-using-mac-address-pool-for-vms
+toc::[]
+
+:FeatureName: KubeMacPool
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+The _KubeMacPool_  component provides a MAC address pool service for virtual machines in designated namepsaces. Enable a MAC address pool in a namespace by applying the KubeMacPool label to that namespace.
+
+include::modules/virt-about-kubemacpool.adoc[leveloffset=+1]
```

Comment:
all these links are dead also.
will it be automatically fixed when the PR is merged and the path is there?

Reply:
As above